rancher2/0_provider_upgrade_test.go

```diff
@@ -427,15 +427,21 @@ func testAccRancher2UpgradeRancher() resource.TestCheckFunc {
 
 func testAccRancher2UpgradeVars() resource.TestCheckFunc {
 	return func(s *terraform.State) error {
-		for _, rs := range s.RootModule().Resources {
+		for k, rs := range s.RootModule().Resources {
 			if rs.Type != "rancher2_bootstrap" {
 				continue
 			}
-
 			token := rs.Primary.Attributes["token"]
-			os.Setenv("RANCHER_TOKEN_KEY", token)
+			rancherTokenKey := os.Getenv("RANCHER_TOKEN_KEY")
+			rancherAdminPass := os.Getenv("RANCHER_ADMIN_PASS")
+			if err := os.Setenv("RANCHER_TOKEN_KEY", token); err != nil {
+				fmt.Printf("Failed to update RANCHER_TOKEN_KEY on ressource %s with err %s", k, err.Error())
```

Comment:
Nit: `resource` instead of `ressource`. Fix all instances and clean up error messages if we are going to commit this